app/code/Inchoo/Sample04/view/frontend/templates/comment/view.phtml

```diff
@@ -0,0 +1,25 @@
+<?php
+/** @var \Magento\Framework\View\Element\Template $block */
+/** @var \Magento\Framework\Escaper $escaper */
+
+/** @var \Inchoo\Sample04\ViewModel\Comment $commentViewModel */
+$commentViewModel = $block->getData('comment');
+?>
+
+<?php if ($comment = $commentViewModel->getComment()) : ?>
+    <p>Created: <?= $escaper->escapeHtml($news->getData('created_at')) ?></p>
+    <p>Updated: <?= $escaper->escapeHtml($news->getData('updated_at')) ?></p>
+    <p>Content: <?= /** @noEscape */ $news->getData('content') ?></p>
+    <p>Comments:
+    <?php foreach ($commentViewModel->getCommentList($news->getId()) as $comment) : ?>
```

Comment:
što će ti ostali komentari ovdje? to ti je bolje dodati ispod svakog news na news view page